esmf_regrid/esmf_regridder.py

```diff
@@ -79,11 +82,21 @@ def __init__(self, src, tgt, precomputed_weights=None):
         self.src = src
         self.tgt = tgt
 
+        if method == "conservative":
+            esmf_regrid_method = ESMF.RegridMethod.CONSERVE
+        elif method == "bilinear":
+            esmf_regrid_method = ESMF.RegridMethod.BILINEAR
+        else:
+            raise ValueError("method must be either bilinear or conservative.")
+        self.method = method
+
         self.esmf_regrid_version = esmf_regrid.__version__
         if precomputed_weights is None:
             self.esmf_version = ESMF.__version__
             weights_dict = _get_regrid_weights_dict(
-                src.make_esmf_field(), tgt.make_esmf_field()
+                src.make_esmf_field(),
+                tgt.make_esmf_field(),
+                regrid_method=esmf_regrid_method,
```

Comment:
I don't think that works here, `esmf_regrid_method` is an ESMF object while `self.method` is a string. The reason we hold `self.method` as a string is because it's used in saving/loading.

Reply:
Oops! Cheerfully withdrawn